pkg/inject/oauth_proxy.go

```diff
@@ -33,16 +33,36 @@ func OAuthProxy(jaeger *v1.Jaeger, dep *appsv1.Deployment) *appsv1.Deployment {
 	return dep
 }
 
-func getOAuthProxyContainer(jaeger *v1.Jaeger) corev1.Container {
+func proxyInitArguments(jaeger *v1.Jaeger) []string {
+	secret, err := util.GenerateProxySecret()
+	if err != nil {
+		jaeger.Logger().Warnf("Error generating secret: %s, fallback to fixed secret", secret)
+		secret = "ncNDoqLGrayxXzxTn5ANbOXZp3qXd0LA"
```

Comment:
Could you please explain this? What this secret stands for?

Reply:
nit export it to a constant with a comment what does it stand for

Reply:
Secret should be something random, but in case of failure I just fallback to something fixed, now the operator uses the word "SECRET"